MediaBrowser.Api/VideosService.cs

```diff
@@ -138,32 +138,28 @@ public void Post(MergeVersions request)
             var videosWithVersions = items.Where(i => i.MediaSourceCount > 1)
                 .ToList();
 
-            var primaryVersion = videosWithVersions.FirstOrDefault();
-
-            if (primaryVersion == null)
-            {
-                primaryVersion = items.OrderBy(i =>
-                    {
-                        if (i.Video3DFormat.HasValue)
-                        {
-                            return 1;
-                        }
-
-                        if (i.VideoType != Model.Entities.VideoType.VideoFile)
-                        {
-                            return 1;
-                        }
-
-                        return 0;
-                    })
-                    .ThenByDescending(i =>
-                    {
-                        var stream = i.GetDefaultVideoStream();
-
-                        return stream == null || stream.Width == null ? 0 : stream.Width.Value;
-
-                    }).First();
-            }
+            var primaryVersion = videosWithVersions.FirstOrDefault()
+                                 ?? items.OrderBy(
+                                         i =>
+                                         {
+                                             if (i.Video3DFormat.HasValue)
+                                             {
+                                                 return 1;
+                                             }
+                                             else
```

Comment:
Why add the additional indentation here? I think the previous version was better.

Though, if we're making a change here it might as well be a ternary like this
```
bool sortFirst = i.Video3DFormat.HasValue || i.VideoType != Model.Entities.VideoType.VideoFile;
return sortFirst ? 1 : 0;
```

Reply:
done